### PR TITLE
Gdb 7265 configure which languages to be indexed by the fts add fts repo configs

### PR DESCRIPTION
**Pull request by MartinaDimova**

### CODE REVIEW

File: src/pages/graphs-visualizations.html [hunk 1]
```diff
@@ -149,7 +149,7 @@ <h5>
                                     <samp class="text-muted small text-overflow d-block" ng-if="config.startMode == 'query'">{{config.startGraphQuery}}</samp>
                                     <samp class="text-muted small text-overflow d-block" ng-if="config.startMode == 'node'">{{config.startIRI}}</samp>
                                 </td>
-                                <td class="text-xs-right" ng-if="isUser() && config.owner === principal().username">
+                                <td class="text-xs-right" ng-if="isUser() && data.config.owner === data.principal().username">
```
Comment (boboetuk) [inline src/pages/graphs-visualizations.html:152]: What is the need for this change?

File: test-cypress/fixtures/locale-en.json [hunk 2]
```diff
@@ -963,6 +963,8 @@
     "repoTooltips.throwQueryEvaluationExceptionOnTimeout": "Repository throws QueryEvaluationException when the duration of a query execution exceeds the timeout parameter.",
     "repoTooltips.queryLimitResults": "Sets the maximum number of results returned from a query after which the evaluation of a query will be terminated; values less than or equal to zero mean no limit.",
     "repoTooltips.nonInterpretablePredicates": "Semicolon-delimited list of predicates (full URIs) that GraphDB will not try to process with the registered GraphDB plugins.",
+    "repoTooltips.fts": "Enable full-text search.",
+    "repoTooltips.ftsLanguages": "Sets languages and respectively the appropriate analyzers for enabled full-text search.",
```
Comment (boboetuk) [inline test-cypress/fixtures/locale-en.json:967]: White space-delimited list of language that should have specific index with appropriate analyzer for full-text search.

File: src/i18n/locale-en.json [hunk 3]
```diff
@@ -963,6 +965,8 @@
     "repoTooltips.throwQueryEvaluationExceptionOnTimeout": "Repository throws QueryEvaluationException when the duration of a query execution exceeds the timeout parameter.",
     "repoTooltips.queryLimitResults": "Sets the maximum number of results returned from a query after which the evaluation of a query will be terminated; values less than or equal to zero mean no limit.",
     "repoTooltips.nonInterpretablePredicates": "Semicolon-delimited list of predicates (full URIs) that GraphDB will not try to process with the registered GraphDB plugins.",
+    "repoTooltips.fts": "Enable full-text search.",
+    "repoTooltips.ftsLanguages": "White space-delimited list of language that should have specific index with appropriate analyzer for full-text search.",
```
Comment (boboetuk) [inline src/i18n/locale-en.json:969]: I think the `language` above should be in plural form

File: src/i18n/locale-en.json [hunk 4]
```diff
@@ -537,6 +537,8 @@
     "menu.sparql.label": "SPARQL",
     "menu.monitor.label": "Monitor",
     "menu.queries.and.updates.label": "Queries and Updates",
+    "menu.fts.label": "Enable full-text search",
```
Comment (teodossidossev) [inline src/i18n/locale-en.json:540]: did you forget to move this translations (lines 540 and 541) to the test fixture?

**Comment by teodossidossev** [comment]
lgtm

**Comment by desislava-hristova-ontotext** [comment]
Jenkins retry build

**Comment by MartinaDimova** [comment]
Jenkins retry build

**Comment by MartinaDimova** [comment]
Jenkins retry build

**Comment by MartinaDimova** [comment]
Jenkins retry build

**Comment by MartinaDimova** [comment]
Jenkins retry build

**Review by desislava-hristova-ontotext — APPROVED** [review]

**Pull request merged**